src/org/exist/backup/ZipArchiveBackupDescriptor.java

```diff
@@ -42,6 +44,8 @@
 
 public class ZipArchiveBackupDescriptor extends AbstractBackupDescriptor {
 
+    private final static Logger LOG = LogManager.getLogger();
```

Comment:
Don't we need a class here for the logger? e.g. `getLogger(ZipArchiveBackupDescriptor.class)`?

Reply:
AFAIK this is not needed (anymore). in the era of log4j 1.2.x is was indeed required for some time.

see https://logging.apache.org/log4j/2.x/log4j-api/apidocs/org/apache/logging/log4j/LogManager.html#getLogger()